src/Distribution/Nixpkgs/Haskell/FromCabal/Name.hs

```diff
@@ -147,6 +147,7 @@ libNixName "taglib_c"                           = return "taglib"
 libNixName "tensorflow"                         = return "libtensorflow"
 libNixName "udev"                               = return "systemd";
 libNixName "uuid"                               = return "libossp_uuid";
+libNixName "vte-2.91"                           = return "vte-2.91"
```

Comment:
This will result in a Nix expression with invalid syntax since a `.` is not allowed in a Nix identifier:

```
$ nix eval '(let vte-2.91 = "foo"; in vte-2.91)'
error: syntax error, unexpected FLOAT, expecting '.' or '=', at (string):1:11
```

Reply:
Introducing a fake name like "vte_291" might work. So:

```haskell
libNixName "vte-2.91" = return "vte_291"
```

```haskell
    resolveInNixpkgs i
      ...
      | i == "vte_291" = binding # (i, path # ["pkgs","gnome3","vte"])
```

I'm not sure this is a hack but I think it might work.

Reply:
Would moving vte to nixpkgs top-level help? I think we will want to do that eventually anyway.

Reply:
@basvandijk I was under the impression that it would be okay because I was overriding it in both [`resolveInNixPkgs`](https://github.com/NixOS/cabal2nix/pull/368/files#diff-500289182850937444c5f7f86d609268R137) and [`libNixName`](https://github.com/NixOS/cabal2nix/pull/368/files#diff-cd05d955733f4bc3562fec4757d99eaeR150).

But I'll go ahead and change it just in case!

Reply:
FYI, you can have dots in attribute names if they're quoted: `{ "foo.bar" = null; }`